### ISSUE TITLE
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'poseidon' defined in class path resource [web.xml]: Unsatisfied dependency expressed through constructor parameter 2: Ambiguous argument values for parameter of type [com.flipkart.poseidon.rotation.RotationCheckServlet] - did you specify the correct bean references as arguments?

### ISSUE THREAD

**Issue by allwinsr** [flipkart-incubator/Poseidon#204]
I'm getting below Exception while starting the application. Can you plz help me to resolve this?

Exception : org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'poseidon' defined in class path resource [web.xml]: Unsatisfied dependency expressed through constructor parameter 2: Ambiguous argument values for parameter of type [com.flipkart.poseidon.rotation.RotationCheckServlet] - did you specify the correct bean references as arguments?

Logs:
```************************************
______ 
| ___ \
| |_/ /
|  __/ 
| |    Handlers Initialized.
\_|    Now starting Poseidon...
************************************
2022-02-07 13:18:53,162 WARN org.springframework.context.support.ClassPathXmlApplicationContext: Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'poseidon' defined in class path resource [web.xml]: Unsatisfied dependency expressed through constructor parameter 2: Ambiguous argument values for parameter of type [com.flipkart.poseidon.rotation.RotationCheckServlet] - did you specify the correct bean references as arguments?
Fatal error in bootstrap sequence. Cannot continue!
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'poseidon' defined in class path resource [web.xml]: Unsatisfied dependency expressed through constructor parameter 2: Ambiguous argument values for parameter of type [com.flipkart.poseidon.rotation.RotationCheckServlet] - did you specify the correct bean references as arguments?
        at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:785)
        at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:228)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1358)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1204)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
        at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:882)
        at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
        at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:144)
        at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:85)
        at com.flipkart.poseidon.BootstrapListener.startPoseidon(BootstrapListener.java:73)
        at com.flipkart.poseidon.BootstrapListener.onApplicationEvent(BootstrapListener.java:56)
        at com.flipkart.poseidon.BootstrapListener.onApplicationEvent(BootstrapListener.java:34)
        at org.trpr.platform.runtime.impl.event.BootstrapProgressMonitor.onApplicationEvent(BootstrapProgressMonitor.java:92)
        at org.trpr.platform.runtime.impl.event.BootstrapProgressMonitor.onApplicationEvent(BootstrapProgressMonitor.java:1)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:403)
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:409)
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:360)
        at org.trpr.platform.core.impl.event.PlatformEventProducerImpl.publishEvent(PlatformEventProducerImpl.java:50)
        at com.flipkart.phantom.runtime.impl.spring.ServiceProxyComponentContainer.publishEvent(ServiceProxyComponentContainer.java:297)
        at com.flipkart.phantom.runtime.impl.spring.ServiceProxyComponentContainer.publishBootstrapEvent(ServiceProxyComponentContainer.java:305)
        at org.trpr.platform.runtime.impl.container.spring.SpringContainerImpl.publishBootstrapEvent(SpringContainerImpl.java:117)
        at org.trpr.platform.runtime.impl.bootstrap.spring.Bootstrap.publishBootstrapEvent(Bootstrap.java:324)
        at org.trpr.platform.runtime.impl.bootstrap.spring.Bootstrap.start(Bootstrap.java:222)
        at org.trpr.platform.runtime.impl.bootstrap.spring.Bootstrap.init(Bootstrap.java:135)
        at com.flipkart.poseidon.Poseidon.main(Poseidon.java:117)
** Trooper runtime shutdown initiated....**
** Trooper runtime Stopping....**
2022-02-07 13:18:53,163 ERROR org.trpr.platform.runtime.impl.bootstrap.spring.Bootstrap: Fatal error in bootstrap sequence. Cannot continue!
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'poseidon' defined in class path resource [web.xml]: Unsatisfied dependency expressed through constructor parameter 2: Ambiguous argument values for parameter of type [com.flipkart.poseidon.rotation.RotationCheckServlet] - did you specify the correct bean references as arguments?
        at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:785) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:228) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1358) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1204) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:882) ~[org.springframework.spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878) ~[org.springframework.spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550) ~[org.springframework.spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:144) ~[org.springframework.spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:85) ~[org.springframework.spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at com.flipkart.poseidon.BootstrapListener.startPoseidon(BootstrapListener.java:73) ~[com.flipkart.poseidon.container-6.0.0-BETA.jar:?]
        at com.flipkart.poseidon.BootstrapListener.onApplicationEvent(BootstrapListener.java:56) ~[com.flipkart.poseidon.container-6.0.0-BETA.jar:?]
        at com.flipkart.poseidon.BootstrapListener.onApplicationEvent(BootstrapListener.java:34) ~[com.flipkart.poseidon.container-6.0.0-BETA.jar:?]
        at org.trpr.platform.runtime.impl.event.BootstrapProgressMonitor.onApplicationEvent(BootstrapProgressMonitor.java:92) ~[org.trpr.runtime-core-2.0.1.jar:?]
        at org.trpr.platform.runtime.impl.event.BootstrapProgressMonitor.onApplicationEvent(BootstrapProgressMonitor.java:1) ~[org.trpr.runtime-core-2.0.1.jar:?]
        at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172) ~[org.springframework.spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165) ~[org.springframework.spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139) ~[org.springframework.spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:403) ~[org.springframework.spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:409) ~[org.springframework.spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:360) ~[org.springframework.spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
        at org.trpr.platform.core.impl.event.PlatformEventProducerImpl.publishEvent(PlatformEventProducerImpl.java:50) ~[org.trpr.platform-core-2.0.1.jar:?]
        at com.flipkart.phantom.runtime.impl.spring.ServiceProxyComponentContainer.publishEvent(ServiceProxyComponentContainer.java:297) ~[com.flipkart.phantom.runtime-3.8.0.jar:?]
        at com.flipkart.phantom.runtime.impl.spring.ServiceProxyComponentContainer.publishBootstrapEvent(ServiceProxyComponentContainer.java:305) ~[com.flipkart.phantom.runtime-3.8.0.jar:?]
        at org.trpr.platform.runtime.impl.container.spring.SpringContainerImpl.publishBootstrapEvent(SpringContainerImpl.java:117) ~[org.trpr.runtime-core-2.0.1.jar:?]
        at org.trpr.platform.runtime.impl.bootstrap.spring.Bootstrap.publishBootstrapEvent(Bootstrap.java:324) ~[org.trpr.runtime-core-2.0.1.jar:?]
        at org.trpr.platform.runtime.impl.bootstrap.spring.Bootstrap.start(Bootstrap.java:222) ~[org.trpr.runtime-core-2.0.1.jar:?]
        at org.trpr.platform.runtime.impl.bootstrap.spring.Bootstrap.init(Bootstrap.java:135) ~[org.trpr.runtime-core-2.0.1.jar:?]
        at com.flipkart.poseidon.Poseidon.main(Poseidon.java:117) ~[com.flipkart.poseidon.container-6.0.0-BETA.jar:?]

```

**Comment by regunathb** [comment]
Some more information would help. Were you able to run the examples as-is as detailed here : https://github.com/flipkart-incubator/Poseidon/tree/master/sample#poseidon-sample ? Are you running Poseidon along with some other app/runtime that is pulling in different versions of dependencies?

**Comment by allwinsr** [comment]
Hi @regunathb / @pmohankumar ,

I've tried with the master branch.

Were you able to run the examples as-is as detailed here : https://github.com/flipkart-incubator/Poseidon/tree/master/sample#poseidon-sample ? 
--> No, I tried as-is(followed the same steps mentioned) that is what failing.

Are you running Poseidon along with some other app/runtime that is pulling in different versions of dependencies?
--> No, I just tried to run this app on the terminal after cloning the repo.

``` mvn dependency:tree
[INFO] Scanning for projects...
[INFO] 
[INFO] --------------------< com.flipkart.poseidon:sample >--------------------
[INFO] Building Poseidon Sample 5.13.0
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ sample ---
[INFO] com.flipkart.poseidon:sample:jar:5.13.0
[INFO] +- com.flipkart.poseidon:api:jar:5.13.0:compile
[INFO] |  +- com.flipkart.poseidon:core:jar:5.13.0:compile
[INFO] |  |  +- com.flipkart.lego:model:jar:3.1.0:compile
[INFO] |  |  +- com.flipkart.lego:core:jar:3.1.0:compile
[INFO] |  |  +- javax.servlet:javax.servlet-api:jar:3.1.0:compile
[INFO] |  |  +- com.netflix.hystrix:hystrix-core:jar:1.5.8:compile
[INFO] |  |  |  +- com.netflix.archaius:archaius-core:jar:0.4.1:compile
[INFO] |  |  |  |  \- commons-configuration:commons-configuration:jar:1.8:compile
[INFO] |  |  |  +- io.reactivex:rxjava:jar:1.2.0:compile
[INFO] |  |  |  \- org.hdrhistogram:HdrHistogram:jar:2.1.9:compile
[INFO] |  |  +- com.google.guava:guava:jar:18.0:compile
[INFO] |  |  +- org.slf4j:slf4j-api:jar:1.7.9:compile
[INFO] |  |  +- javax.inject:javax.inject:jar:1:compile
[INFO] |  |  +- com.github.kristofa:brave-impl:jar:2.2.1:compile
[INFO] |  |  |  +- com.github.kristofa:brave-interfaces:jar:2.2.1:compile
[INFO] |  |  |  |  \- org.apache.thrift:libthrift:jar:0.9.0:compile
[INFO] |  |  |  +- org.apache.commons:commons-lang3:jar:3.4:compile
[INFO] |  |  |  \- org.slf4j:jcl-over-slf4j:jar:1.7.5:compile
[INFO] |  |  +- com.codahale.metrics:metrics-core:jar:3.0.2:compile
[INFO] |  |  +- com.codahale.metrics:metrics-healthchecks:jar:3.0.2:compile
[INFO] |  |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.11.2:compile
[INFO] |  |  |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.11.2:compile
[INFO] |  |  |  \- com.fasterxml.jackson.core:jackson-core:jar:2.11.2:compile
[INFO] |  |  \- com.flipkart.poseidon:common:jar:5.13.0:compile
[INFO] |  +- com.flipkart.poseidon:container:jar:5.13.0:compile
[INFO] |  |  +- com.flipkart.poseidon:log4j-access:jar:5.13.0:compile
[INFO] |  |  +- commons-fileupload:commons-fileupload:jar:1.3.3:compile
[INFO] |  |  |  \- commons-io:commons-io:jar:2.7:compile
[INFO] |  |  +- org.springframework:spring-context:jar:5.2.5.RELEASE:compile
[INFO] |  |  |  +- org.springframework:spring-aop:jar:5.2.5.RELEASE:compile
[INFO] |  |  |  +- org.springframework:spring-core:jar:5.2.5.RELEASE:compile
[INFO] |  |  |  |  \- org.springframework:spring-jcl:jar:5.2.5.RELEASE:compile
[INFO] |  |  |  \- org.springframework:spring-expression:jar:5.2.5.RELEASE:compile
[INFO] |  |  +- org.springframework:spring-web:jar:5.2.5.RELEASE:compile
[INFO] |  |  +- org.springframework:spring-beans:jar:5.2.5.RELEASE:compile
[INFO] |  |  +- org.hibernate:hibernate-validator:jar:5.4.2.Final:compile
[INFO] |  |  |  +- javax.validation:validation-api:jar:1.1.0.Final:compile
[INFO] |  |  |  +- org.jboss.logging:jboss-logging:jar:3.3.0.Final:compile
[INFO] |  |  |  \- com.fasterxml:classmate:jar:1.3.1:compile
[INFO] |  |  +- org.eclipse.jetty:jetty-server:jar:9.4.44.v20210927:compile
[INFO] |  |  |  +- org.eclipse.jetty:jetty-http:jar:9.4.44.v20210927:compile
[INFO] |  |  |  \- org.eclipse.jetty:jetty-io:jar:9.4.44.v20210927:compile
[INFO] |  |  +- org.eclipse.jetty:jetty-servlet:jar:9.4.44.v20210927:compile
[INFO] |  |  |  +- org.eclipse.jetty:jetty-security:jar:9.4.44.v20210927:compile
[INFO] |  |  |  \- org.eclipse.jetty:jetty-util-ajax:jar:9.4.44.v20210927:compile
[INFO] |  |  +- org.eclipse.jetty:jetty-rewrite:jar:9.4.44.v20210927:compile
[INFO] |  |  +- org.eclipse.jetty:jetty-servlets:jar:9.4.44.v20210927:compile
[INFO] |  |  |  +- org.eclipse.jetty:jetty-continuation:jar:9.4.44.v20210927:compile
[INFO] |  |  |  \- org.eclipse.jetty:jetty-util:jar:9.4.44.v20210927:compile
[INFO] |  |  +- org.apache.logging.log4j:log4j-jcl:jar:2.17.1:compile
[INFO] |  |  |  \- commons-logging:commons-logging:jar:1.2:compile
[INFO] |  |  +- com.lmax:disruptor:jar:3.3.4:compile
[INFO] |  |  +- com.codahale.metrics:metrics-servlets:jar:3.0.2:compile
[INFO] |  |  |  +- com.codahale.metrics:metrics-json:jar:3.0.2:compile
[INFO] |  |  |  \- com.codahale.metrics:metrics-jvm:jar:3.0.2:compile
[INFO] |  |  +- com.codahale.metrics:metrics-jetty9:jar:3.0.2:compile
[INFO] |  |  +- com.netflix.hystrix:hystrix-metrics-event-stream:jar:1.5.8:compile
[INFO] |  |  |  \- com.netflix.hystrix:hystrix-serialization:jar:1.5.8:runtime
[INFO] |  |  |     +- com.fasterxml.jackson.module:jackson-module-afterburner:jar:2.7.5:runtime
[INFO] |  |  |     \- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.7.5:runtime
[INFO] |  |  +- com.netflix.hystrix:hystrix-codahale-metrics-publisher:jar:1.4.0-RC5:compile
[INFO] |  |  \- org.jetbrains:annotations:jar:13.0:compile
[INFO] |  +- com.flipkart.hydra:hydra-dispatcher:jar:1.3:compile
[INFO] |  |  +- com.flipkart.hydra:hydra-composer:jar:1.3:compile
[INFO] |  |  |  \- com.flipkart.hydra:hydra-expression:jar:1.3:compile
[INFO] |  |  \- com.flipkart.hydra:hydra-task:jar:1.3:compile
[INFO] |  +- com.google.code.findbugs:jsr305:jar:3.0.1:compile
[INFO] |  \- org.mockito:mockito-all:jar:1.10.19:compile
[INFO] +- com.flipkart.poseidon:sampleSC:jar:5.13.0:compile
[INFO] |  \- com.flipkart.poseidon:service-clients-core:jar:5.13.0:compile
[INFO] |     +- com.flipkart.phantom:task:jar:3.8.0:compile
[INFO] |     |  +- org.trpr:platform-core:jar:2.0.1:compile
[INFO] |     |  |  +- org.trpr:platform-model:jar:2.0.1:compile
[INFO] |     |  |  +- org.springframework:spring-context-support:jar:5.2.5.RELEASE:compile
[INFO] |     |  |  +- org.springframework:spring-tx:jar:5.2.5.RELEASE:compile
[INFO] |     |  |  +- org.perf4j:perf4j:jar:0.9.15:compile
[INFO] |     |  |  +- commons-lang:commons-lang:jar:2.4:compile
[INFO] |     |  |  +- org.bouncycastle:bcprov-jdk16:jar:1.45:compile
[INFO] |     |  |  \- org.bouncycastle:bcprov-ext-jdk16:jar:1.45:compile
[INFO] |     |  +- com.github.kristofa:brave-zipkin-spancollector:jar:2.2.1:compile
[INFO] |     |  +- org.apache.httpcomponents:httpclient:jar:4.5.13:compile
[INFO] |     |  |  \- commons-codec:commons-codec:jar:1.11:compile
[INFO] |     |  \- org.apache.httpcomponents:httpcore:jar:4.3:compile
[INFO] |     \- com.flipkart.poseidon:http-handler:jar:5.13.0:compile
[INFO] |        \- org.apache.httpcomponents:httpmime:jar:4.3.6:compile
[INFO] \- com.flipkart.phantom:sample-task-proxy:jar:3.8.0:compile
[INFO]    +- com.flipkart.phantom:runtime:jar:3.8.0:compile
[INFO]    |  +- org.trpr:runtime-core:jar:2.0.1:compile
[INFO]    |  +- junit:junit:jar:4.13.1:test
[INFO]    |  |  \- org.hamcrest:hamcrest-core:jar:1.3:test
[INFO]    |  +- org.springframework:spring-webmvc:jar:5.2.5.RELEASE:compile
[INFO]    |  +- org.freemarker:freemarker:jar:2.3.30:compile
[INFO]    |  +- org.eclipse.jetty:jetty-webapp:jar:9.4.44.v20210927:compile
[INFO]    |  |  \- org.eclipse.jetty:jetty-xml:jar:9.4.44.v20210927:compile
[INFO]    |  \- io.netty:netty:jar:3.3.1.Final:compile
[INFO]    +- org.apache.logging.log4j:log4j-api:jar:2.17.1:compile
[INFO]    +- org.apache.logging.log4j:log4j-core:jar:2.17.1:compile
[INFO]    +- org.apache.logging.log4j:log4j-slf4j-impl:jar:2.17.1:compile
[INFO]    \- org.apache.logging.log4j:log4j-1.2-api:jar:2.16.0:compile
```

**Comment by pmohankumar** [comment]
ack, debugging this

**Comment by pmohankumar** [comment]
Fixed this issue now (an earlier change #158 missed out updating sample application). Please pull the latest master and follow the steps in running [Sample](https://github.com/flipkart-incubator/Poseidon/tree/master/sample#to-run-from-command-line)
